pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -353,13 +360,24 @@ var _ = Describe("vmexport", func() {
 					"BlockOwnerDeletion": HaveValue(BeFalse()),
 				})), "owner ref BlockOwnerDeletion should be false for secret",
 			)
-		})
+		},
+			Entry("using PVC source", vmexport.PVC_FLAG, "PersistentVolumeClaim"),
+			Entry("using Snapshot source", vmexport.SNAPSHOT_FLAG, "VirtualMachineSnapshot"),
+			Entry("using VM source", vmexport.VM_FLAG, "VirtualMachine"),
+		)
 
-		It("VirtualMachineExport is deleted succesfully", func() {
-			_, err := virtClient.ExportV1beta1().VirtualMachineExports(metav1.NamespaceDefault).Create(context.Background(), vme, metav1.CreateOptions{})
-			Expect(err).ToNot(HaveOccurred())
+		DescribeTable("Delete command runs successfully", func(exists bool) {
+			if exists {
```

Comment:
Why it creates a new VME if it exists?

Reply:
These are two cases, delete a VME that exists and delete non-existing VME. The command should not fail in both cases.